### PR TITLE
Role Selection

### LINKED ISSUE #51: Degree/Year selection
When a user joins the server, they should be prompted to select their degree as well as their enrollment year (e.g. 2020/2021).

To do this, use Discord's new selection components, which allows up to 25 items (should be enough). If it isn't due to the amount of degrees, separate undergraduate and master degrees.

Degrees that have both undergraduate and master, should be in the same option (e.g. L/MEIC-A).

Don't forget to add a "TurISTa" role and its selection ~~(should be given automatically when a user joins the server)~~ (see @D-Card 's comment below)

When a user selects an option, it should clear other degree roles they may have and then add the corresponding role. According to the degree, the user might also be given the "Alameda" or "Tagus Park" role. The same should happen for enrollment year (the year roles should be called something like "Ano 1 (2020/2021)").

In this initial development phase, keep configuration in a static JSON file. In the future, the course list/year list might be configured through slash commands, stored on SQLite and/or fetched from Fénix's API.

### PR DESCRIPTION
**Pull request by RafDevX**
Closes #51

### CODE REVIEW

File: src/prisma/schema.prisma [hunk 1]
```diff
@@ -1,6 +1,6 @@
 datasource db {
     provider = "sqlite"
-    url      = "file:./data/bot.db"
+    url      = env("DATABASE_URL")
```
Comment (diogotcorreia) [inline src/prisma/schema.prisma:3]: There is no documentation on README on how to setup this on `docker-compose.yml`.

Possibly
```yml
build:
  context: .
  args:
    DATABASE_URL: "file:/app/data/bot.db"
```

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,6 @@
 FROM node:16.6.1-alpine3.14
+ARG DATABASE_URL
+ENV DATABASE_URL ${DATABASE_URL}
```
Comment (diogotcorreia) [inline Dockerfile:3]: Even with this, migrations aren't automatically applied when the Docker container starts!

**Review by diogotcorreia — CHANGES_REQUESTED** [review]

File: src/modules/populate.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+							label: "LEFT",
+							description:
+								"Licenciatura em Engenharia Física e Tecnológica",
+							value: "876961271667372073",
+							emoji: "⚛️",
+						},
+					],
+				},
+			},
+		});
```
[the remark this answers is not in the document]
Comment (RafDevX) [reply src/modules/populate.ts:41]: It's mock data for testing, the whole populate module probably shouldn't even be called in production and probably shouldn't even be in version control but I'm not sure how to organize this.

File: Dockerfile (hunk 2 above)
Comment (RafDevX) [reply Dockerfile:3]: ¯\_(ツ)_/¯

File: README.md [hunk 4]
```diff
@@ -11,13 +11,21 @@ version: "3.8"
...
+			args:
+				DATABASE_URL: file:./data/bot.db
+		## END;
         volumes:
             - type: bind
               source: ./data
               target: /app/data
         environment:
             DISCORD_TOKEN: PLACE_BOT_TOKEN_HERE
+			GUILD_ID: PLACE_MAIN_GUILD_ID_HERE # or "GLOBAL" to use in multiple guilds (1hr roll-out time)
```
Comment (diogotcorreia) [inline README.md:28]: change all to spaces

**Review by diogotcorreia — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (RafDevX) [reply README.md]: fixed

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**